.gitignore

```diff
@@ -3,3 +3,4 @@
 /composer.lock
 /coverage*
 /phpunit.xml
+/.idea
```

Comment:
Sorry for not being clear enough, I meant adding 
```suggestion
/.php-cs-fixer.php
```

Reply:
I didn't see the old file there so I didn't get it.